src/commands/Configuration/configure/main.ts

```diff
@@ -0,0 +1,111 @@
+				},
+				{
+					label: await client.bulbutils.translate("config_main_options.logging", interaction.guild?.id, {}),
+					value: "logging",
+					description: await client.bulbutils.translate("config_main_options_descriptions.logging", interaction.guild?.id, {}),
+					emoji: Emotes.configure.LOGGING,
+				},
+				{
+					label: await client.bulbutils.translate("config_main_options.prefix", interaction.guild?.id, {}),
+					value: "prefix",
+					description: await client.bulbutils.translate("config_main_options_descriptions.prefix", interaction.guild?.id, {}),
+					emoji: Emotes.features.MEMBER_LIST_DISABLED,
+				},
+				{
+					label: await client.bulbutils.translate("config_main_options.quick_reasons", interaction.guild?.id, {}),
+					value: "quickReasons",
+					description: await client.bulbutils.translate("config_main_options_descriptions.quick_reasons", interaction.guild?.id, {}),
+					emoji: Emotes.features.MEMBER_PROFILES,
+				},
+				{
+					label: await client.bulbutils.translate("config_main_options.roles_on_leave", interaction.guild?.id, {}),
+					value: "rolesOnLeave",
+					description: await client.bulbutils.translate("config_main_options_descriptions.roles_on_leave", interaction.guild?.id, {}),
+					emoji: Emotes.features.HUB,
+				},
+				{
+					label: await client.bulbutils.translate("config_main_options.timezone", interaction.guild?.id, {}),
+					value: "timezone",
+					description: await client.bulbutils.translate("config_main_options_descriptions.timezone", interaction.guild?.id, {}),
+					emoji: Emotes.configure.TIMEZONE,
+				},
+			]),
+	);
+
+	await interaction.update({
+		content: await client.bulbutils.translate("config_main_header", interaction.guild?.id, {
+			guild: interaction.guild,
+		}),
+		components: [row],
+	});
+
+	const filter = i => i.user.id === interaction.user.id;
+	const collector = interaction.channel?.createMessageComponentCollector({ filter, time: 60000, max: 1 });
+
+	collector?.on("collect", async (i: MessageComponentInteraction) => {
+		if (i.isSelectMenu()) {
+			switch (i.values[0]) {
+				case "actionsOnInfo":
+					await require("./actionsOnInfo").default(i, client);
+					break;
+				case "automod":
+					await require("./automod").default(i, client);
+					break;
+				case "autorole":
+					await require("./autorole").default(i, client);
+					break;
+				case "language":
+					await require("./language").default(i, client);
+					break;
+				case "logging":
+					await require("./logging").default(i, client);
+					break;
+				case "prefix":
+					await require("./prefix").default(i, client);
+					break;
+				case "quickReasons":
+					await require("./quickReasons").default(i, client);
+					break;
+				case "rolesOnLeave":
+					await require("./rolesOnLeave").default(i, client);
+					break;
+				case "timezone":
+					await require("./timezone").default(i, client);
+					break;
+			}
```

Comment:
```suggestion
			await require(`./configure/${i.values[0]}`).default(i, client);
```

Reply:
Invalid relative path to the module when you attempt to run it.

Reply:
Lol. Now it's ok, I edited.